tests/foreman/ui/test_errata.py

```diff
@@ -156,13 +156,13 @@ def test_positive_list(self):
         with Session(self.browser) as session:
             session.nav.go_to_errata()
             self.errata.show_only_applicable(False)
-            self.assertIsNone(self.errata.search(CUSTOM_REPO_ERRATA_ID))
-            self.assertIsNotNone(self.errata.search(FAKE_1_ERRATA_ID))
+            self.assertIsNone(self.errata.search(FAKE_3_ERRATA_ID))
```

Comment:
I tried to answer it in my comment [above](https://github.com/SatelliteQE/robottelo/pull/3858#discussion_r80494703) (1 min before yours comment so i suspect github didn't show you it in time).
Just saw the errata ids are outdated because of different repos we use now, carefully checked what's actually available in the first and the second organization and updated the test with corresponding changes.